generate-docs/script-inputs/local-repo-snippets.yaml

```diff
@@ -1798,6 +1798,39 @@ Office.AddinCommands.Source#id:member:
 
         event.completed();
     }
+Office.AsyncContextOptions:interface:
+  - |-
+    // The following example gets the attachment contents of the
+    // current mail item being composed in Outlook.
+    function getAttachmentContentCompose() {
+        const item = Office.context.mailbox.item;
+        const options = { asyncContext: { currentItem: item } };
+        item.getAttachmentsAsync(options, callback);
+
+        function callback(result) {
```

Comment:
How much of this function is required to know how the callback options work? I don't mind the full sample, but I wonder if the function contents could be `//...` and get the same point across. (Though maybe I'm oversimplifying this and missing something.)

Reply:
I initially went with that option, but determined that the comments didn't help simplify the sample. Having the full callback function still helped to show how to store callback options and use them.